pyani_plus/private_cli.py

```diff
@@ -766,9 +768,21 @@ def anib(  # noqa: PLR0913
         .one()
         .length
     )
-    subject_stem = Path(hash_to_filename[subject_hash]).stem
     outfmt = "6 " + " ".join(method_anib.BLAST_COLUMNS)
 
+    if hash_to_filename[subject_hash].endswith(".gz"):
+        # We must decompress the FASTA file for makeblastdb
+        subject_fasta = tmp_dir / hash_to_filename[subject_hash][:-3]
+        subject_stem = subject_fasta.stem
+        with (
+            gzip.open(fasta_dir / hash_to_filename[subject_hash], "rb") as f_in,
+            subject_fasta.open("wb") as f_out,
+        ):
+            shutil.copyfileobj(f_in, f_out)
```

Comment:
This use of ``shutil.copyfileobj`` is taken from the https://docs.python.org/3/library/gzip.html documentation.